File: Expert Setter.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int y;
        float x,a;
        scanf("%f%d",&x,&y);
        a=(y/x)*100;
        if(a>=50)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: Expert Setter.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            if((a/2.0)<=b)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}